### ISSUE TITLE
coco_cap dataset

### ISSUE THREAD

**Issue by mattgithub1919** [intersun/LightningDOT#7]
Hello, 

Just a quick question. As far as I can see, coco_cap dataset is not shared in lightningDOT repo. I have been able to download coco_cap dataset from Uniter repo (with ./scripts/download_indomain.sh). May I know if I can use the coco_cap dataset from Uniter for lightningDOT training? Kindly let me know. Thank you.

**Comment by intersun** [comment]
Yep, you can use it directly. The pre-training dataset for this paper and UNITER are **exactly** the same.
